Content.Client/Clown/ClownShoesSystem.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Content.Shared.Clown;
+using Content.Shared.Inventory.Events;
+
+namespace Content.Client.Clown;
+
+public sealed class ClownShoesSystem : EntitySystem
```

Comment:
should be WaddleClothingSystem in a clothing namespace

Reply:
Missed that rename. Should be WaddleSystem. Fixed.

Would prefer to keep this in a Clown namespace, as it's a Clown Waddle not a Clothing Waddle.

Reply:
anyone can wear clown shoes not just the clown, and in the future i want to give ducky slippers a small waddle too

Reply:
Content.Client.Clothing.Systems.Clown is where I've put them. I think waddling around is clowning about :D